nuxeo-ai-core/src/main/java/org/nuxeo/ai/enrichment/AbstractEnrichmentService.java

```diff
@@ -101,4 +113,19 @@ public String saveJsonAsRawBlob(String rawJson) {
     protected List<TagSuggestion> asTags(List<AIMetadata.Tag> tags) {
         return Collections.singletonList(new TagSuggestion(suggestionProperty, tags));
     }
+
+    protected KeyValueStore getStore() {
+        KeyValueService kvs = Framework.getService(KeyValueService.class);
+        return kvs.getKeyValueStore("default");
+    }
+
+    protected void storeCallback(KeyValueStore store, String jobId, Map<String, Serializable> params) {
+        try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
+             ObjectOutputStream oos = new ObjectOutputStream(baos)) {
+            oos.writeObject(params);
+            store.put(jobId, baos.toByteArray(), TimeUnit.DAYS.toMillis(1));
```

Comment:
You are hardcoding the store timeout to `TimeUnit.DAYS.toMillis(1)`.  Is that the best value?

Reply:
if there is no response for more than one day than we have troubles

Reply:
of course we can set it to 30 days just to be sure. anyway the value will be released when the endpoint is reached. still, don't think the value should be configurable